back/engines/free/email_campaigns/app/models/email_campaigns/campaigns/moderator_digest.rb

```diff
@@ -148,7 +148,7 @@ def statistics(project)
             increase: participants_increase,
             past_increase: participants_past_increase
           },
-          total_participants: ps.project_participants(project).size
+          total_participants: ps.projects_participants([project]).size
```

Comment:
Haven't changed this digest to include anonymous participants and everyone surveys but have changed the total to use the non-cached version given that the increases on line 125/126 use the non cached method, otherwise there could be some discrepancies.